build.gradle

```diff
@@ -307,7 +307,6 @@ configure(allprojects) { project ->
 
 	tasks.withType(AbstractArchiveTask) {
 		preserveFileTimestamps = false
-		reproducibleFileOrder = true
 	}
```

Comment:
This whole block should be unnecessary. With `@Classpath` and `@CompileClasspath` inputs Gradle normalizes away both the file order and the timestamps in JAR files.

Reply:
I removed the whole block. After discussion, it's not worth doing this just to get the `:spring-core:jar` up-to-date.